eva-pipeline/src/test/java/embl/ebi/variation/eva/pipeline/jobs/AnnotationConfig.java

```diff
@@ -0,0 +1,26 @@
+package embl.ebi.variation.eva.pipeline.jobs;
+
+import org.springframework.batch.test.JobLauncherTestUtils;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
+
+/**
+ * @author Diego Poggioli
+ *
+ * Configuration init for Annotation Job
+ */
+@Configuration
+@PropertySource({"annotation.properties"})
+public class AnnotationConfig {
+    @Bean
+    public static PropertySourcesPlaceholderConfigurer propertySourcesPlaceholderConfigurer() {
+        return new PropertySourcesPlaceholderConfigurer();
+    }
+
+    @Bean
+    public JobLauncherTestUtils jobLauncherTestUtils() {
+                return new JobLauncherTestUtils();
+           }
```

Comment:
`JobLauncherTestsUtils.class` can be added to the @ContextConfiguration in the JUnit test class, making these @Bean method unnecessary.